apache/superset
# Bump nvd3 charts for tooltip fix

### CATEGORY

Choose one


### SUMMARY

Fixes erroneous removal of tooltips for charts on dashboards. see details at https://github.com/apache-superset/superset-ui-plugins/pull/167

### TEST PLAN

Use a dashboard with multiple tabs:
1. hover over a chart on tab 1
2. switch to tab 2
3. resize the window, then resize it back to the original size
4. switch back to tab 1
5. still see tooltips on the previously hovered chart

### ADDITIONAL INFORMATION


- [x] Has associated issue: Might be related to #7822

### REVIEWERS
@graceguo-supercat @kristw @xtinec
